testkit/src/main/scala/org/bitcoins/testkit/core/gen/CurrencyUnitGenerator.scala

```diff
@@ -36,6 +36,12 @@ trait CurrencyUnitGenerator {
   def feeUnit: Gen[FeeUnit] =
     Gen.oneOf(satsPerByte, satsPerKiloByte, satsPerVirtualByte)
 
+  def feeUnit(maxFee: Long): Gen[FeeUnit] = {
+    Gen.choose(0L, maxFee / 10000L).map { n =>
```

Comment:
why divide by `10000L` here?

Reply:
`maxFee` represents the max fee that would be okay for a given transaction  so I can't allow anything near maxFee to be the rate for a single kilobyte, instead I divide by a big enough number that no issues occur

Reply:
added scaladoc saying what `maxFee` is